src/fidesctl/api/ctl/migrations/versions/7c851d8a102a_add_created_at_and_updated_at.py

```diff
@@ -16,40 +16,45 @@
 branch_labels = None
 depends_on = None
 
+SQL_MODEL_LIST = [
+    "data_categories",
+    "data_qualifiers",
+    "data_uses",
+    "data_subjects",
+    "datasets",
+    "evaluations",
+    "policies",
+    "registries",
+    "systems",
+    "organizations",
+]
+
 
 def upgrade():
-    for k, model in sql_model_map.items():
```

Comment:
@SteveDMurphy I remember specifically praising you for this clever solution, and the universe remembered too, and decided to serve me some irony 😂 took me like 2 hours to figure out why it was breaking